docs/create-packages/Sign-a-Package.md

```diff
@@ -0,0 +1,101 @@
+---
+title: How to Create a Localized NuGet Package | Microsoft Docs
+author: rido-min
+ms.author: rido-min
+manager: unniravindranathan
+ms.date: 02/23/2018
+ms.topic: article
+ms.prod: nuget
+ms.technology: null
+description: Explains how signed packages can be used to enable content integrity verification
+keywords: NuGet package signing, NuGet security, creating signed packages
+ms.reviewer:
+- karann-msft
+- anangaur
+---
+
+# Signed packages
+
+NuGet packages can include a digital signature which provides protection against tampered content. This signature is produced from an x509 certificate that also adds authenticity proofs to the actual origin of the package. 
+
+> [!Warning] 
+> NuGet.org does not accept signed packages yet. This feature will be announced using the [announcements](https://github.com/NuGet/Announcements) github repository.
```

Comment:
When will this happen? Do we want to publish this page before nuget.org is ready? Can someone make use of signed packages before that time via a private feed?

Reply:
we don't have a final date yet. And yes, users can sign packages and consume them from private feeds.